### PR TITLE
Prep psmr

### PR DESCRIPTION
**Pull request by johannesmayer**
Updated the download scripts for PET and MR data.
MR checks for an existence of the GRAPPA .zip archive.

Both force an overwrite of already existing files when unzipping an archive.
Running them twice will hence not result in the need for user input.

Also continued working on notebooks and saw that there is no more cell output is in the notebooks.

### CODE REVIEW

**Comment by johannesmayer** [comment]
I updated the importing of matplotlib based on the Azure client tests displaying the figures inline now.

**Comment by KrisThielemans** [comment]
i haven't followed the conversation but normally `%matplotlib notebook` should be best (should allow you to resize etc)

File: notebooks/MR/interactive/a_fully_sampled.ipynb [hunk 1]
```diff
@@ -40,7 +40,8 @@
     "##   distributed under the License is distributed on an \"AS IS\" BASIS,\n",
     "##   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
     "##   See the License for the specific language governing permissions and\n",
-    "##   limitations under the License."
+    "##   limitations under the License.\n",
+    "%matplotlib inline"
```
Comment (casperdcl) [inline notebooks/MR/interactive/a_fully_sampled.ipynb]: `%matplotlib notebook` is better

File: notebooks/MR/interactive/b_kspace_filter.ipynb [hunk 2]
```diff
@@ -39,7 +39,8 @@
     "##   distributed under the License is distributed on an \"AS IS\" BASIS,\n",
     "##   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
     "##   See the License for the specific language governing permissions and\n",
-    "##   limitations under the License."
+    "##   limitations under the License.\n",
+    "%matplotlib inline"
```
Comment (casperdcl) [inline notebooks/MR/interactive/b_kspace_filter.ipynb]: same

File: notebooks/MR/interactive/c_coil_combination.ipynb [hunk 3]
```diff
@@ -39,7 +39,8 @@
     "##   distributed under the License is distributed on an \"AS IS\" BASIS,\n",
     "##   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
     "##   See the License for the specific language governing permissions and\n",
-    "##   limitations under the License."
+    "##   limitations under the License.\n",
+    "%matplotlib inline"
```
Comment (casperdcl) [inline notebooks/MR/interactive/c_coil_combination.ipynb]: same

File: notebooks/MR/interactive/d_undersampled_reconstructions.ipynb [hunk 4]
```diff
@@ -40,8 +40,7 @@
     "##   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
     "##   See the License for the specific language governing permissions and\n",
     "##   limitations under the License.\n",
-    "\n",
-    "\n"
+    "%matplotlib inline"
```
Comment (casperdcl) [inline notebooks/MR/interactive/d_undersampled_reconstructions.ipynb]: same

**Comment by KrisThielemans** [comment]
@johannesmayer  make sure you follow the updated instructions for `nbstripout`, then `git fetch` and `git merge origin/master` in here, replace the `matplotlib inline` with `notebook` and let us know.

**Comment by johannesmayer** [comment]
I merged SIRF-Exercises/master, replaced the `inline` by `notebook` and I followed the `INSTALL.md` instructions on the bottom on how to install the script, but could you please confirm that it looks like the script is running well:

When I run a cell, and then save the notebook with output, i.e. figures and the counter brackets to [somenumber], then `git status` shows me the notebook has been modified, but `git diff` does not generate output. `git add` of the notebook followed by `git commit` will say 
```
On branch prep-psmr
nothing to commit, working tree clean
```
Is this good to go?

**Comment by casperdcl** [comment]
sounds right

**Comment by casperdcl** [comment]
just quickly skimmed through some of these notebooks - not sure if you have a reason for inconsistent padding spaces around brackets (i.e. `help( recon )` rather than `help(recon)`) - couldn't see a pattern

**Comment by johannesmayer** [comment]
I agree there is no pattern probably, I would have to clean them up tomorrow

**Pull request merged**